packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -629,7 +629,7 @@ export default class SchedulerTask {
                 status: SchedulerJobStatus.COMPLETED,
             });
             if (process.env.IS_PULL_REQUEST !== 'true' && !payload.isPreview) {
-                this.schedulerClient.generateValidation({
+                await this.schedulerClient.generateValidation({
```

Comment:
this file was a goldmine.. I am not sure if we should await or safely "ignore"

Reply:
Just based on the `generateValidation` method, seems we'd want to await here, even if just for semantics (the method returns a job ID).

Reply:
@ZeRego can you confirm if it's fine to add `await` here? and the rest of the awaits in this file?

Reply:
This should be a `void` or add a `.catch` as we don't want to wait for it.